Mathlib/GroupTheory/OreLocalization/Cardinality.lean

```diff
@@ -0,0 +1,112 @@
+/-
+Copyright (c) 2024 Jz Pan. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jz Pan
+-/
+import Mathlib.GroupTheory.OreLocalization.Basic
+import Mathlib.SetTheory.Cardinal.Arithmetic
+
+/-!
+
+# Cardinality of Ore localizations
+
+This file contains some results on cardinality of Ore localizations.
+
+## TODO
+
+- Prove or disprove `OreLocalization.cardinal_mk_le_lift_cardinal_mk_of_commute`
+  with `Commute` assumption removed.
+
+-/
+
+universe u v
+
+open Cardinal
```

Comment:
Maybe 
```suggestion
open Cardinal Function
```
to avoid writing `Function.Surjective` everywhere?

Reply:
Sure.